test/ReadsAlignmentSet_basic_test.py

```diff
@@ -1,290 +1,289 @@
-        {
-            "ref": alignment_ref,
-            "ref_path_to_set": ref_path_to_set,
-            "include_item_info": 0,
-            "include_set_item_ref_paths": 1,
-        },
-    )[0]
-    assert fetched_set is not None
-    assert "data" in fetched_set
-    assert "info" in fetched_set
-    assert len(fetched_set["data"]["items"]) == n_items
-    assert alignment_ref == info_to_ref(fetched_set["info"])
-    for item in fetched_set["data"]["items"]:
-        assert "info" not in item
-        assert "ref" in item
-        assert "label" in item
-        assert "ref_path" in item
-        assert item["ref_path"] == ";".join(ref_path_to_set) + ";" + item["ref"]
-
-    if DEBUG:
-        log_this(config, "RNASeq_Alignment_with_ref_path_to_set", fetched_set)
-
-
-def test_get_alignment_set(
-    set_api_client: SetAPI,
-    context: dict[str, str | list],
-    ws_id: int,
-    alignment_refs: list[str],
-) -> None:
-    alignment_set_name = "test_alignment_set"
-    alignment_items = [{"label": "wt", "ref": ref} for ref in alignment_refs]
-    n_items = len(alignment_refs)
-    alignment_set = {"description": "test_alignments", "items": alignment_items}
-    alignment_set_ref = set_api_client.save_reads_alignment_set_v1(
-        context,
-        {
-            "workspace_id": ws_id,
-            "output_object_name": alignment_set_name,
-            "data": alignment_set,
-        },
-    )[0]["set_ref"]
+        if params.get(REF_PATH_TO_SET, []) != []:
+            # add in some values for the REF_PATH_TO_SET
+            params[REF_PATH_TO_SET] = [
+                rnaseq_alignment_sets[idx],
+            ]
 
-    fetched_set = set_api_client.get_reads_alignment_set_v1(
-        context, {"ref": alignment_set_ref, "include_item_info": 0}
-    )[0]
-    assert fetched_set is not None
-    assert "data" in fetched_set
-    assert "info" in fetched_set
-    assert len(fetched_set["data"]["items"]) == n_items
-    assert alignment_set_ref == info_to_ref(fetched_set["info"])
-    for item in fetched_set["data"]["items"]:
-        assert "info" not in item
-        assert "ref" in item
-        assert "label" in item
-
-    fetched_set_with_info = set_api_client.get_reads_alignment_set_v1(
-        context, {"ref": alignment_set_ref, "include_item_info": 1}
-    )[0]
-    assert fetched_set_with_info is not None
-    assert "data" in fetched_set_with_info
-    for item in fetched_set_with_info["data"]["items"]:
-        assert "info" in item
-        assert "ref" in item
-        assert "label" in item
+        fetched_set = set_api_client.get_reads_alignment_set_v1(context, params)[0]
 
+        # set items are not returned in order so can't check the second set item
+        check_get_set_output(
+            fetched_set,
+            set_name=set_name,
+            set_type="KBaseRNASeq.RNASeqAlignmentSet",
+            set_description="",
+            n_items=n_items,
+            **params,
+            is_fake_set=True,
+        )
```

Comment:
this tests the other type of set that can be returned by the ReadsAlignment API. It doesn't create the same structure as the genuine reads alignment sets do, so the inputs to the test results checker need to be tweaked somewhat.